packages/driver/src/util/deferred.ts

```diff
@@ -0,0 +1,18 @@
+import Bluebird from 'bluebird'
```

Comment:
Where is this file being used?

Reply:
It's not, it's vestigial. Good catch. Removed in https://github.com/cypress-io/cypress/pull/21137/commits/9e3750b693391bfadeae08c5ba88b888bb6dd6dd